datafusion/expr/src/udf_docs.rs

```diff
@@ -142,6 +143,23 @@ impl DocumentationBuilder {
         self
     }
 
+    /// Add a standard "expression" argument to the documentation
+    ///
+    /// This is a common argument for scalar UDFs that operate on an expression and is rendered like
+    ///
+    /// ```text
+    /// <arg_name>:
+    ///   <expression_type> expression to operate on. Can be a constant, column, or function, and any combination of arithmetic operators.
+    /// ```
+    pub fn with_standard_argument(
```

Comment:
it might be useful to mention how standard argument differs from just argument?

Reply:
Good idea, done in 09a4b7bb0